arch/arm/mach-rockchip/board.c

```diff
@@ -34,16 +36,31 @@ int board_late_init(void)
 	return rk_board_late_init();
 }
 
+#define GPIO_HUB_RESET 149
+#define GPIO_HOST_EN 66
+
 int board_init(void)
 {
 	int ret;
-
```

Comment:
Be careful about introducing unrelated changes. The line removed above has nothing to do with adding Toybrick support

Reply:
Will revert back the removed line.